Makefile

```diff
@@ -40,7 +40,7 @@ ci-run:
 
 ci-passed-tests:
 	echo -n matrix= >> $$GITHUB_OUTPUT
-	amm .github/scripts/ci.sc passedJson $(RUNNERS) .github/passed.txt ./passed.json
+	amm .github/scripts/ci.sc passedJson $(RUNNERS) $(PASSEDFILE) ./passed.json
```

Comment:
This idea is good, but consuming more than one file and scheduling them is better.